test/e2e/cypress/e2e/saptune_details.cy.js

```diff
@@ -0,0 +1,117 @@
+import { saptuneDetailsData } from '../fixtures/saptune-details/saptune_details_data';
+
+describe('Saptune Details page', () => {
+  const { hostID, packageVersion, configuredVersion } = saptuneDetailsData;
+
+  const notFoundContainerSelector = '.pb-24';
+  const versionContainerSelector =
+    '.max-w-7xl > :nth-child(1) > :nth-child(1) > :nth-child(3)';
+  const tuningStatusIcon = ':nth-child(3) > :nth-child(2) > :nth-child(1)';
+  const saptuneServiceStatusSelector =
+    '.max-w-7xl > :nth-child(1) > :nth-child(1) > :nth-child(5)';
+  const saptuneServiceStatusIcon =
+    '.grid-flow-col > :nth-child(1) > :nth-child(2) > :nth-child(1) > .flex';
+  const sapconfServiceStatusIcon =
+    '.grid-flow-col > :nth-child(2) > :nth-child(2) > :nth-child(1) > .flex';
+  const tunedServiceStatusIcon =
+    ':nth-child(3) > :nth-child(2) > :nth-child(1) > .flex';
+  const saptuneTuningSolutionsSelector =
+    '.max-w-7xl > :nth-child(1) > :nth-child(1) > :nth-child(7)';
+  const saptuneTuningNotesSelector =
+    '.max-w-7xl > :nth-child(1) > :nth-child(1) > :nth-child(9)';
+  const saptuneStagingStatusSelector = ':nth-child(11)';
+
+  before(() => {
+    cy.visit(`hosts/${hostID}/saptune`);
+  });
+
+  it('should render saptune details not found if saptune payload version is unsupported', () => {
+    cy.loadScenario('host-vmhdbdev01-saptune-unsupported');
+    cy.get(notFoundContainerSelector).should(
+      'contain',
+      'Saptune Details Not Found'
+    );
+  });
+
+  it('should render default not found if saptune payload is not installed', () => {
+    cy.loadScenario('host-vmhdbdev01-saptune-uninstalled');
+    cy.get(notFoundContainerSelector).should(
+      'contain',
+      'Saptune Details Not Found'
+    );
+  });
+
+  it('should render versions list view when saptune is not tuning correctly', () => {
+    cy.loadScenario('host-vmhdbdev01-saptune-not-tuned');
+    cy.get(versionContainerSelector)
+      .should('contain', 'Package')
+      .should('contain', packageVersion)
+      .should('contain', 'Configured Version')
+      .should('contain', configuredVersion)
+      .should('contain', 'Tuning')
+      .should('contain', 'No tuning');
+    cy.get(`${tuningStatusIcon} svg`)
+      .should('exist')
+      .should('have.class', 'fill-yellow-500');
+  });
+
+  it('should render each part of saptune service status with green passing icons', () => {
+    cy.loadScenario('host-vmhdbdev01-saptune-service-status-passing');
```

Comment:
I don't see this test really needed. We could test the text in the previous test.
In any case, if you want to keep it, I would remove the icon testing